### DISCUSSION TITLE
The intuition behind multiplying by 114?

### DISCUSSION THREAD

**Discussion by dsantiago** [Megvii-BaseDetection/YOLOX#983] [Q&A] [answered]
Just trying to understand, thanks!

https://github.com/Megvii-BaseDetection/YOLOX/blob/dd5700c24693e1852b55ce0cb170342c19943d8b/yolox/data/data_augment.py#L146

**Comment by FateScript** [comment, marked as answer]
It's not really matters what value you are using in padding. The value here  comes from a legacy problem. You could set it to 0 or 255 or any value you like, your model could fit this value after enough training.

**Reply by dsantiago** [reply]
Thanks for your time answering. Understood it now, never sure when there's some hidden shenanigans.
